packages/core/src/components/sections/ProductShelf/DefaultComponents.ts

```diff
@@ -1,6 +1,20 @@
-import { ProductShelf as UIProductShelf } from '@faststore/ui'
-import ProductCard from 'src/components/product/ProductCard'
-import Carousel from 'src/components/ui/Carousel'
+import dynamic from 'next/dynamic'
+const UIProductShelf = dynamic(() =>
+  /* webpackChunkName: "UIProductShelf" */
+  import('@faststore/ui').then((mod) => ({ default: mod.ProductShelf }))
```

Comment:
question: why is the `default: ` necessary here?

Reply:
I think it's not necessary hahah
I streamlined it [here](https://github.com/vtex/faststore/pull/2559/commits/d7311750d3b54397e278970b9b83d4262a45faea)